src/main/kotlin/racing/CarRaciongMain.kt

```diff
@@ -0,0 +1,23 @@
+package racing
+
+class Main {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
+            startCarRacing()
+        }
+    }
+}
```

Comment:
Java처럼 main 함수를 만들 때, 꼭 Class를 선언할 필요가 없어요. args 메서드 파라미터도 생략할 수 있어요. 😉
```suggestion
fun main() {
    startCarRacing()
}
```

Reply:
우왕 최고 깔끔해졌네요 👏🏻